app/src/pages/trace/SpanFeedback.tsx

```diff
@@ -36,29 +42,14 @@ const columns = [
   },
 ];
 
-export function SpanEvaluationsTable(props: {
-  span: SpanEvaluationsTable_evals$key;
+function SpanAnnotationsTable({
+  annotations,
+}: {
+  annotations: Mutable<SpanFeedback_annotations$data["spanAnnotations"]>;
```

Comment:
Is the mutable necessary? Technically read only is correct.

Reply:
its necessary for passing it in to the table, i can spread it in the table so that the prop doesn't get mutated by it. That's probably a better call

Reply:
Even though unnecessary I think it's better to be safe with the types for now.